tests/test_indexing.py

```diff
@@ -19,7 +19,10 @@ def test_index():
 def test_full_window():
     with rasterio.open('tests/data/RGB.byte.tif') as src:
         left, bottom, right, top = src.bounds
-        assert src.window(left, bottom, right, top) == tuple(zip((0, 0), src.shape))
+        assert src.window(
```

Comment:
@eseglem I think this makes it less easy to read.  Lines 21-25 could collapse down to: 

```
assert src.window(*src.bounds) == tuple(zip((0, 0), src.shape))
```